source/includes/orders.md

```diff
@@ -273,6 +273,7 @@ store_id | yes | Store where the order will be created
 user | yes | The user object (see below)
 scheduled_time | no | The date and time the order is scheduled to be fulfilled, if not present or null the assumptions is an ASAP order
 status_board_display_name | no | Name that will be displayed on the status board to inform the customer of their order. If not supplied the name will be infered from the user's first and last name.
+unlock_code | no | An alphanumeric code that will be used to open a pickup device. If present on the order, the pickup device will be locked and require the code entered in order to open
```

Comment:
Strongly would ask this to be modeled with more structure than just a single key.

Reply:
Updated!